### PR TITLE
UIDATIMP-217: add associated job profiles search on action profile de…

### PR DESCRIPTION
**Pull request by OleksandrYehorovEPAM**
…tails pane

### CODE REVIEW

File: src/settings/ActionProfiles/AssociatedJobProfiles.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
+import React, {
+  Fragment, useEffect,
```
Comment (VictorSoroka1) [inline src/settings/ActionProfiles/AssociatedJobProfiles.js]:
```suggestion
  Fragment,
  useEffect,
```

File: src/settings/ActionProfiles/AssociatedJobProfiles.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                          onClick={e => e.stopPropagation()}
+                        >
+                          <Checkbox
+                            name="selected-all"
+                            checked={isAllSelected}
+                            onChange={handleSelectAllCheckbox}
+                          />
+                        </div>
+                      ),
+                      name: intl.formatMessage({ id: 'ui-data-import.name' }),
```
Comment (VictorSoroka1) [inline src/settings/ActionProfiles/AssociatedJobProfiles.js]: Just for curiosity, I think now columns strings can be passed as FormattedMessage components. Though it still requires a check.

File: src/settings/ActionProfiles/AssociatedJobProfiles.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+                      tags: intl.formatMessage({ id: 'ui-data-import.tags' }),
+                      updated: intl.formatMessage({ id: 'ui-data-import.updated' }),
+                      updatedBy: intl.formatMessage({ id: 'ui-data-import.updatedBy' }),
+                    }}
+                    columnWidths={columnWidths}
+                    formatter={associatedJobProfilesFormatter}
+                    onHeaderClick={onSort}
```
Comment (VictorSoroka1) [inline src/settings/ActionProfiles/AssociatedJobProfiles.js]: Please move callback to the bottom.

File: src/settings/ActionProfiles/AssociatedJobProfiles.js (hunk 2 above)
Comment (OleksandrYehorovEPAM) [reply src/settings/ActionProfiles/AssociatedJobProfiles.js]: That is true. Changed it everywhere.

Comment (VictorSoroka1) [reply src/settings/ActionProfiles/AssociatedJobProfiles.js]: Great.

**Pull request merged**